ui/app/scripts/controllers/AuthenticationCtrl.js

```diff
@@ -4,11 +4,34 @@
 (function() {
     'use strict';
     angular.module('theHiveControllers')
-        .controller('AuthenticationCtrl', function($scope, $state, $uibModalStack, AuthenticationSrv, NotificationSrv) {
+        .controller('AuthenticationCtrl', function($scope, $state, $location, $uibModalStack, $stateParams, AuthenticationSrv, NotificationSrv, UtilsSrv, UrlParser, appConfig) {
             $scope.params = {};
 
             $uibModalStack.dismissAll();
 
+            $scope.ssoLogin = function (code) {
+                AuthenticationSrv.ssoLogin(code, function(data, status, headers) {
+                    var redirectLocation = headers().location;
+                    if(angular.isDefined(redirectLocation)) {
```

Comment:
@saibot94 Could you explain what does this line? I mean: I understand that it takes the `Location` header of the response and redirect to it, but I don't understand why.